packages/blockchain/tron/src/lib/services/tron.trc20.ts

```diff
@@ -18,22 +18,21 @@ const prepareSignedTransaction = async (body: TransferTronTrc20, tronWeb: ITronW
 
   const client = tronWeb.getClient(provider)
 
-  const destinationContractAddress = getProxyTokenAddress(tokenAddress)
-
-  client.setAddress(destinationContractAddress)
-  const contractInstance = await client.contract().at(destinationContractAddress)
-  const decimals = await contractInstance.decimals().call()
+  client.setAddress(tokenAddress)
+  const contractInstance = await client.contract(Trc20Token.abi, tokenAddress)
```

Comment:
@alexloiko Did you tried it? It was not working for me. Also you need to do get balance of.

Reply:
yes I've tested it on mainnet, it works. will doublecheck on testnet